ocs_ci/ocs/bucket_utils.py

```diff
@@ -3006,3 +3014,71 @@ def get_obj_versions(mcg_obj, awscli_pod, bucket_name, obj_key):
             d["ETag"] = d["ETag"].strip('"')
 
     return versions_dicts
+
+
+def compare_object_versions(mcg_obj, awscli_pod, bucket_a, bucket_b, obj_key):
```

Comment:
Not important but I would suggest to rename `bucket_a` and `bucket_b` with some other names like `source_bucket`, 'destination_bucket' or `bucket_one` and `bucket_two`

Reply:
`source_bucket` and `destination_bucket` are too specific. For example, in test_bidirectional_replication_with_versioning` both buckets could be considered source and dest.

However, I'll replace `bucket_a` with `first_bucket` and `bucket_b` with `second_bucket` to make the methods more readable.

Reply:
Fixed in https://github.com/red-hat-storage/ocs-ci/pull/11438/commits/0eda7a886d95a7d462b578683e127fc71500d443